### ISSUE TITLE
tensorflow版本的NER

### ISSUE THREAD

**Issue by ArtificialIdoit** [buppt/ChineseNER#16]
作者有遇到过所有字的预测准确率很高，但是一个命名实体都抽取不出来的预测结果吗？

**Comment by buppt** [comment]
> 作者有遇到过所有字的预测准确率很高，但是一个命名实体都抽取不出来的预测结果吗？

...字的准确率应该在98%以上吧，毕竟有好多不属于实体的字来提高准确率

**Comment by ArtificialIdoit** [comment]
> > 作者有遇到过所有字的预测准确率很高，但是一个命名实体都抽取不出来的预测结果吗？
> 
> ...字的准确率应该在98%以上吧，毕竟有好多不属于实体的字来提高准确率

找到问题了，是因为我修改了超参数，learning rate=1e-4，batch_size=256造成的。
